AppCenterAuth/AppCenterAuthTests/MSAuthTests.m

```diff
@@ -1383,7 +1383,7 @@ - (void)testRefreshWithExpiredTokenWhileMsalNotConfigured {
   [authTokenContextMock stopMocking];
 }
 
-- (void)testRefreshRetriedAfterMsalConfigured {
+- (void)testSignOutIfRefreshWithNoMsalConfiguration {
```

Comment:
Why are you replacing the unit test?

Reply:
Yeah this seems kind of confusing, is this not changing the intent of the test case?  Do we not need both?

Reply:
@jaelim-ms That was the case we discussed where we should actually sign out in that case because in the valid flow that couldn't happen.